.github/workflows/notify-on-label.yml

```diff
@@ -1,19 +1,29 @@
 # Notify users/teams when labels are added to an issue.
 
-name: Notify when adding label to issue
+name: Notify FE team for FE label on issues
```

Comment:
The downside of this approach is that we have to create a new workflow for each notify job now - instead of using one. I think this is fine, since it's a bit more explicit, but the alternative would be to just add another job instead in this file instead of creating a whole new workflow

Reply:
agreed... I made a not further down that I was aiming to reflect reality and hope people will just change the name, but a note up here would be good too.  I'll add a new PR